core/process.py

```
@@ -65,12 +71,30 @@ def start_pipecat_process(
+
     # Start the process
     process = subprocess.Popen(
         command,
         env=os.environ.copy(),  # Copy the current environment
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        text=True,  # Capture output as text
     )
 
+    # Log stdout and stderr asynchronously
+    async def log_output():
```

Pylint (3.3.7)

[warning] Unused variable 'log_output'

(W0612)